client/src/containers/Home/Home.jsx

```diff
@@ -2,70 +2,70 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Dimmer, Loader, Image, Segment } from 'semantic-ui-react';
 import styles from './Home.module.css';
 import FloatingActionButton from '../../components/FloatingActionButton/FloatingActionButton';
 import BlogCard from '../../components/BlogCard/index';
 import '../../customStyles.css';
 import HomeExtended from './HomeExtended';
 import useGetTopBlogs from '../../hooks/useGetTopBlogs';
+import HomeLoader from './HomeLoader';
 
 const getBlog = (blog, size) => {
   const user = blog.userId && blog.userId.name ? blog.userId : {};
   return (
     <div className={styles[size]} key={blog._id}>
-      <BlogCard {...blog} {...user} href={`/blog/${blog._id}`} />
+      <BlogCard {...user} {...blog} href={`/blog/${blog._id}`} />
     </div>
   );
 };
 
-const blogsLoader = () => (
-  <>
-    <Segment>
-      <Dimmer active>
-        <Loader size="huge">Loading</Loader>
-      </Dimmer>
-
-      <Image src="https://react.semantic-ui.com/images/wireframe/short-paragraph.png" />
-      <Image src="https://react.semantic-ui.com/images/wireframe/short-paragraph.png" />
-      <Image src="https://react.semantic-ui.com/images/wireframe/short-paragraph.png" />
-    </Segment>
-  </>
-);
-
 const Home = ({ isAuthenticated }) => {
   const [blogs = [], fetched] = useGetTopBlogs(1);
 
   const biggerCards = blogs
-    .slice(0, 2)
-    .map(blog => getBlog(blog, 'large-blog'));
+    .slice(0, 4)
+    .map(blog => getBlog(blog, 'large-blog')); 
   const smallerCardsFirstColumn = blogs
-    .slice(2, 6)
+    .slice(4, 8)
     .map(blog => getBlog(blog, 'small-blog'));
   const smallerCardsSecondColumn = blogs
-    .slice(6)
+    .slice(8,12)
     .map(blog => getBlog(blog, 'small-blog'));
 
+  const sampleBlogCard = () => <BlogCard />
```

Comment:
Can you shuffle the array randomly if possible? It might be fun to see blogs getting random order?